include/deal.II/base/point.h

```diff
@@ -22,6 +22,12 @@
 #include <deal.II/base/exceptions.h>
 #include <deal.II/base/tensor.h>
 
+#ifdef DEAL_II_WITH_BOOST
```

Comment:
is this ever false?

Reply:
With the current configuration and with our current dependency model `DEAL_II_WITH_BOOST` is always defined/true.

Reply:
Indeed, we don't use `#ifdef DEAL_II_WITH_BOOST` a single time in the codebase :-D

Reply:
Haha..that's why it was hard to find! ok!